expan/core/experiment.py

```diff
@@ -76,6 +75,10 @@ def delta(self, method='fixed_horizon', kpi_subset=None, derived_kpis=None,
         'bayes_factor':			self.bayes_factor_delta()
         'bayes_precision':		self.bayes_precision_delta()
         """
+        # Check the correct type of the kpi_subset which should be a list
+        if not isinstance(kpi_subset, list) and not all(isinstance(kpi, str) for kpi in kpi_subset):
+            raise TypeError('KPI subset should be a list of strings')
+
```

Comment:
There are no stupid questions =) This is a check whether kpi_subset should be a list of strings because a user can provide a string of a kpi to delta if there is only 1 kpi he want to use, but he should provide a list with 1 kpi, otherwise test fails. This was also added on backend request.